### PR TITLE
Dv/devel emoticons

### PR DESCRIPTION
**Pull request by danielbernalb**
1. Added optional Emoticons and status text for Air Quality levels in every board: miniESP32, Wemos, Heltec and TTGO_TQ with the comment of the option in Platformio.ini :
-D Emoticons
-D NOemoticons
2. Changes in visualization.

### CODE REVIEW

File: lib/gui-utils/src/GUIIcons.h [hunk 1]
```diff
@@ -62,4 +62,85 @@ static const unsigned char PROGMEM ic_sensor_live[] = {
     B10101010,
     B01010100,
     B00000000,
-};
\ No newline at end of file
+};
+
+#ifdef Emoticons
+  static unsigned char Smileface1[] = {                                     //32x32
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIIcons.h:68]: Please change to expression name. The numbers 1, 2, 3 does not mean nothing 
```suggestion
  static unsigned char SmilefaceHappy[] = {                                     //32x32
```
for example.

File: lib/gui-utils/src/GUIIcons.h [hunk 2]
```diff
@@ -62,4 +62,85 @@ static const unsigned char PROGMEM ic_sensor_live[] = {
     B10101010,
     B01010100,
     B00000000,
-};
\ No newline at end of file
+};
+
+#ifdef Emoticons
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIIcons.h:67]: The directives of preprocessor should be in upper case

```suggestion
#ifdef EMOTICONS
```

File: lib/gui-utils/src/GUIUtils.cpp [hunk 3]
```diff
@@ -93,103 +118,292 @@ void GUIUtils::displayEndLine(String msg) {
...
+    u8g2.print("GOOD");
+  }
+  else if (avarage < 36)
+  {
+    u8g2.setFont(u8g2_font_unifont_t_emoticons);
+    u8g2.drawGlyph(76, 12, 0x0062);
+    u8g2.setFont(u8g2_font_4x6_tf);
+    u8g2.setCursor(77, 17);
+    u8g2.print("MODERATE");
+  }
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIUtils.cpp:140]: please specialize it in two methods for not have repeated code, for example something like that:
```suggestion
  {
    displayEmoticonLabel(0x0032,"UNHEALT S")
  }
  else if (avarage < 36)
  {
    displayEmoticonLabel(76,12,0x0062,"MODERATE");
  }
```

File: lib/gui-utils/src/GUIUtils.cpp [hunk 4]
```diff
@@ -93,103 +118,292 @@ void GUIUtils::displayEndLine(String msg) {
...
+  delay(1);
+#endif
+#else
+  if (avarage < 13)
+  {
+#ifdef TTGO_TQ
+    u8g2.drawXBM(1, 0, 32, 32, Smileface1);
+    u8g2.setFont(u8g2_font_6x12_tf);
+    u8g2.setCursor(40, 15);
+    u8g2.print("BUENO/");
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIUtils.cpp:183]: We have two lenguajes here? spanish and english? I think that we have many sections that need possible support for multiple lenguajes, and for size reasons maybe we only should support to english lenguaje

File: lib/gui-utils/src/GUIUtils.cpp [hunk 5]
```diff
@@ -93,103 +118,292 @@ void GUIUtils::displayEndLine(String msg) {
...
+    u8g2.print("MODERADO/");
+    u8g2.setFont(u8g2_font_6x10_tf);
+    u8g2.setCursor(92, 16);
+    u8g2.print("ama");
+#else
+    u8g2.drawXBM(0, 1, 32, 32, Smileface2);
+    u8g2.setFont(u8g2_font_5x7_tf);
+    u8g2.setCursor(25, 29);
+    u8g2.print("MODERADO");
+#endif
```
Comment (hpsaturn) [inline lib/gui-utils/src/GUIUtils.cpp:209]: please specialize it in two methods for not have repeated code.

File: lib/gui-utils/src/GUIIcons.h (hunk 1 above)
Comment (danielbernalb) [reply lib/gui-utils/src/GUIIcons.h:68]: Resolved

Comment (danielbernalb) [reply lib/gui-utils/src/GUIIcons.h]: Expression name changed:
https://github.com/danielbernalb/esp32-hpma115s0/commit/e15cc4bab6ad5edeec8961e4e06e03d533c9c7d5

File: lib/gui-utils/src/GUIIcons.h (hunk 2 above)
Comment (danielbernalb) [reply lib/gui-utils/src/GUIIcons.h]: Directives of preprocessor in upper case:
https://github.com/danielbernalb/esp32-hpma115s0/commit/d78dabae9444e2f0b8892aacb3804c0599a01315

File: lib/gui-utils/src/GUIUtils.cpp (hunk 4 above)
Comment (danielbernalb) [reply lib/gui-utils/src/GUIUtils.cpp]: Resolved:
https://github.com/danielbernalb/esp32-hpma115s0/commit/e15cc4bab6ad5edeec8961e4e06e03d533c9c7d5

File: lib/gui-utils/src/GUIUtils.cpp (hunk 3 above)
Comment (danielbernalb) [reply lib/gui-utils/src/GUIUtils.cpp]: Resolved:
https://github.com/danielbernalb/esp32-hpma115s0/commit/7c5910c159f0f4f362b51306db3957029769818b

File: lib/gui-utils/src/GUIUtils.cpp (hunk 5 above)
Comment (danielbernalb) [reply lib/gui-utils/src/GUIUtils.cpp]: Resolved in branch:
https://github.com/danielbernalb/esp32-hpma115s0/tree/dv/devel_emoticons2

**Comment by danielbernalb** [comment]
All changes now are in the branch:
https://github.com/danielbernalb/esp32-hpma115s0/tree/dv/devel_emoticons2

**Pull request merged**